perfdash/Makefile

```diff
@@ -19,7 +19,7 @@ run: perfdash
 		--address=0.0.0.0:8080 \
 		--builds=20 \
 		--force-builds \
-		--githubConfigDir=https://api.github.com/repos/kubernetes/perf-tests/contents/perfdash/test
+		--configPath=test/jobs.yaml
```

Comment:
ok, this changes the cfg for a local binary.

Reply:
yes, just for testing easier :)